docs/guides/syntax.md

```diff
@@ -151,13 +151,43 @@ Optional.
 
 A suffix mapped to a [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes), for example:
 
--   `a` for `:active`
--   `c` for `:checked`
--   `f` for `:focus`
--   `h` for `:hover`
--   etc.
-
-<p class="noteBox info">You can find the complete list of pseudo-classes and their abbreviations in <a href="https://github.com/acss-io/atomizer/blob/main/packages/atomizer/src/lib/grammar.js#L7">Atomizer&#39;s grammar library</a>.</p>
+-   `:active` => `:a`,
+-   `:checked` => `:c`,
+-   `:default` => `:d`,
+-   `:disabled` => `:di`,
+-   `:empty` => `:e`,
+-   `:enabled` => `:en`,
+-   `:nth-child(even)` => `:even`,
+-   `:first` => `:fi`,
+-   `:first-child` => `:fc`,
+-   `:first-of-type` => `:fot`,
+-   `:fullscreen` => `:fs`,
+-   `:focus` => `:f`,
+-   `:focus-within` => `:fw`,
+-   `:focus-visible` => `:fv`,
+-   `:hover` => `:h`,
+-   `:indeterminate` => `:ind`,
+-   `:in-range` => `:ir`,
+-   `:invalid` => `:inv`,
+-   `:last-child` => `:lc`,
+-   `:last-of-type` => `:lot`,
+-   `:left` => `:l`,
+-   `:link` => `:li`,
+-   `:nth-child(odd)` => `:odd`,
+-   `:only-child` => `:oc`,
+-   `:only-of-type` => `:oot`,
+-   `:optional` => `:o`,
+-   `:out-of-range` => `:oor`,
+-   `:placeholder-shown` => `:ps`,
+-   `:read-only` => `:ro`,
+-   `:read-write` => `:rw`,
+-   `:required` => `:req`,
+-   `:right` => `:r`,
+-   `:root` => `:rt`,
+-   `:scope` => `:s`,
+-   `:target` => `:t`,
+-   `:valid` => `:va`,
+-   `:visited` => `:vi`,
```

Comment:
Would this be easier digested as a table

Reply:
Hmm perhaps, I can try moving it to that

Reply:
fixed https://github.com/acss-io/atomizer/pull/527/commits/3c5b556fe169d4f946dc290ccdee13964ea82300